src/shared/cache/MolecularProfilesInStudyCache.ts

```diff
@@ -17,13 +17,26 @@ function dataToKey(molecularProfiles: MolecularProfile[], studyId: string) {
 async function fetch(
     studyIds: string[]
 ): Promise<AugmentedData<MolecularProfile[], string>[]> {
-    const profiles: MolecularProfile[] = await client.fetchMolecularProfilesUsingPOST(
+    let profiles: MolecularProfile[] = await client.fetchMolecularProfilesUsingPOST(
         {
             molecularProfileFilter: {
                 studyIds,
             } as MolecularProfileFilter,
         }
     );
+
+    //TODO: remove this block once data is fixed
+    profiles = profiles.map(profile => {
+        if (
+            profile.molecularAlterationType === 'STRUCTURAL_VARIANT' &&
```

Comment:
use AlterationTypeConstants for STRUCTURAL_VARIANT and is there a constant for "SV"?

Reply:
updated.